.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -9,13 +9,14 @@ assignees: ''
 **READ THIS FIRST:** If you are having trouble getting MNE-Python to work with
 your own data, you should ask for help on one of our other channels:
 
-- [email list](https://mail.nmr.mgh.harvard.edu/mailman/listinfo/mne_analysis)
-- [Gitter (chat)](https://gitter.im/mne-tools/mne-python)
+- [GitHub Discussions](https://github.com/mne-tools/mne-python/discussions)
+- [Mailing list](https://mail.nmr.mgh.harvard.edu/mailman/listinfo/mne_analysis)
+- [Gitter](https://gitter.im/mne-tools/mne-python)
```

Comment:
I would just completely remove `gitter` from our docs. Let's try GH discussions 100% and revert later if we need to. It's better to concentrate our efforts / try to migrate fully rather than leave more options